src/main.c

```diff
@@ -524,9 +524,10 @@ static void process_command_line_arguments(char *argv[], char **options)
 		fclose(fp);
 	}
 #endif
-
-	/* read all configurations from a config file */
-	(void)read_config_file(g_config_file, options);
+	if (cmd_line_opts_start == 2) {
```

Comment:
Why this?
It changes the current behavior to automatically use a `civetweb.conf` file in the same directory, if started without any parameter. Is this the intention of this pull request?
This would be an incompatible change. Why is it required?